<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createProjectWidget</class>
 <widget class="QWidget" name="createProjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>429</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>create a project</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_12">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Create new project</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>350</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="filesTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>&amp;Files</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>2</y>
              <width>391</width>
              <height>311</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0">
              <widget class="QGroupBox" name="inputFilesGroupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Input</string>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="annotationInfileLabel">
                  <property name="text">
                   <string>Annotation:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="annotationInfileLineEdit"/>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="findAnnotationInfilePushButton">
                  <property name="text">
                   <string>find</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="readsInfileLabel">
                  <property name="text">
                   <string>Alignments:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="readsInfileLineEdit"/>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="findReadsInfilePushButton">
                  <property name="text">
                   <string>find</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="outputFilesGroupBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="title">
                <string>Output</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="annotationOutfileLabel">
                  <property name="text">
                   <string>Annotation:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="annotationOutfileLineEdit"/>
                </item>
                <item row="0" column="2">
                 <widget class="QPushButton" name="findAnnotationOutfilePushButton">
                  <property name="text">
                   <string>find</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="readsOutfileLabel">
                  <property name="text">
                   <string>Alignments:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="readsOutfileLineEdit"/>
                </item>
                <item row="1" column="2">
                 <widget class="QPushButton" name="findReadsOutfilePushButton">
                  <property name="text">
                   <string>find</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="countsOutfileLabel">
                  <property name="text">
                   <string>Counts:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="countsOutfileLineEdit"/>
                </item>
                <item row="2" column="2">
                 <widget class="QPushButton" name="findCountsOutfilePushButton">
                  <property name="text">
                   <string>find</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="parametersTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>&amp;Parameters</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>391</width>
              <height>311</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <widget class="QGroupBox" name="generalParametersGroupBox">
               <property name="title">
                <string>General</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="6" column="0">
                 <widget class="QLabel" name="multiLabel">
                  <property name="text">
                   <string>Use multiple alignments:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="strandedLabel">
                  <property name="text">
                   <string>Consider strand information:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="strandedCheckBox">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="1">
                 <widget class="QCheckBox" name="biasCheckBox">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="minReadsSpinBox">
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="singleStep">
                   <number>5</number>
                  </property>
                  <property name="value">
                   <number>5</number>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QCheckBox" name="multiCheckBox">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="QLabel" name="biasLabel">
                  <property name="text">
                   <string>Consider 3' bias:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="minReadsLabel">
                  <property name="text">
                   <string>Minimal number of hits:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="antisenseLabel">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Reads antisense to genes:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QCheckBox" name="antisenseCheckBox">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="biasParameterGroupBox">
               <property name="title">
                <string>3' bias</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QLabel" name="maxDistLabel">
                  <property name="text">
                   <string>Maximal distance to 3' end:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="maxDistSpinBox">
                  <property name="maximum">
                   <number>9999</number>
                  </property>
                  <property name="singleStep">
                   <number>50</number>
                  </property>
                  <property name="value">
                   <number>250</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="minBelowMaxDistLabel">
                  <property name="text">
                   <string>Minimal number hits below maximal distance:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="minBelowMaxDistSpinBox">
                  <property name="maximum">
                   <number>1000</number>
                  </property>
                  <property name="singleStep">
                   <number>5</number>
                  </property>
                  <property name="value">
                   <number>0</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="regionTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>&amp;Region</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>391</width>
              <height>311</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QCheckBox" name="regionCheckBox">
               <property name="text">
                <string>use only a subregion</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="subRegionGroupBox">
               <property name="title">
                <string>Subregion</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="regionStartNameLabel">
                  <property name="text">
                   <string>Start with chromosome:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="regionStartNameLineEdit"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="regionStartLabel">
                  <property name="text">
                   <string>at position:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="regionStartSpinBox">
                  <property name="maximum">
                   <number>999999999</number>
                  </property>
                  <property name="singleStep">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="regionEndNameLabel">
                  <property name="text">
                   <string>End with chromosome:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="regionEndNameLineEdit"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="regionEndLabel">
                  <property name="text">
                   <string>at position:</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QSpinBox" name="regionEndSpinBox">
                  <property name="maximum">
                   <number>999999999</number>
                  </property>
                  <property name="singleStep">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="settingsTab">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <attribute name="title">
            <string>&amp;Settings</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>391</width>
              <height>311</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QGroupBox" name="indexSettingsGroupBox">
               <property name="title">
                <string>Index</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="0" column="0">
                 <widget class="QLabel" name="indexStepSizeLabel">
                  <property name="text">
                   <string>Size of index steps:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="indexStepSizeSpinBox">
                  <property name="minimum">
                   <number>1000</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="singleStep">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>10000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QGroupBox" name="bufferSettingsGroupBox">
               <property name="title">
                <string>Buffers</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QLabel" name="bufferSizeBamLabel">
                  <property name="text">
                   <string>Size of buffer (reader):</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="bufferSizeBamSpinBox">
                  <property name="minimum">
                   <number>1000</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="singleStep">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>200000</number>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="bufferSizeMapLabel">
                  <property name="text">
                   <string>Size of buffer (mapper):</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="bufferSizeMapSpinBox">
                  <property name="minimum">
                   <number>1000</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="singleStep">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>200000</number>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="bufferSizeOutLabel">
                  <property name="text">
                   <string>Size of buffer (writer):</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="bufferSizeOutSpinBox">
                  <property name="minimum">
                   <number>1000</number>
                  </property>
                  <property name="maximum">
                   <number>1000000</number>
                  </property>
                  <property name="singleStep">
                   <number>1000</number>
                  </property>
                  <property name="value">
                   <number>200000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="standardButtons">
             <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>annotationInfileLineEdit</tabstop>
  <tabstop>findAnnotationInfilePushButton</tabstop>
  <tabstop>readsInfileLineEdit</tabstop>
  <tabstop>findReadsInfilePushButton</tabstop>
  <tabstop>annotationOutfileLineEdit</tabstop>
  <tabstop>findAnnotationOutfilePushButton</tabstop>
  <tabstop>readsOutfileLineEdit</tabstop>
  <tabstop>findReadsOutfilePushButton</tabstop>
  <tabstop>countsOutfileLineEdit</tabstop>
  <tabstop>findCountsOutfilePushButton</tabstop>
  <tabstop>minReadsSpinBox</tabstop>
  <tabstop>strandedCheckBox</tabstop>
  <tabstop>antisenseCheckBox</tabstop>
  <tabstop>multiCheckBox</tabstop>
  <tabstop>biasCheckBox</tabstop>
  <tabstop>maxDistSpinBox</tabstop>
  <tabstop>minBelowMaxDistSpinBox</tabstop>
  <tabstop>regionCheckBox</tabstop>
  <tabstop>regionStartNameLineEdit</tabstop>
  <tabstop>regionStartSpinBox</tabstop>
  <tabstop>regionEndNameLineEdit</tabstop>
  <tabstop>regionEndSpinBox</tabstop>
  <tabstop>indexStepSizeSpinBox</tabstop>
  <tabstop>bufferSizeBamSpinBox</tabstop>
  <tabstop>bufferSizeMapSpinBox</tabstop>
  <tabstop>bufferSizeOutSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>regionCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>subRegionGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>biasCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>biasParameterGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strandedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>antisenseLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>strandedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>antisenseCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
